packages/InteractiveProfilingTool-Tests.package/IPTTreeModelTestCase.class/instance/testGetIconIfAboveThreshold.st

```diff
@@ -0,0 +1,14 @@
+tests
+testGetIconIfAboveThreshold
+
+	| model messageTally expected actual |
+	
+	model := IPTTreeModelMock new. 
+	messageTally := MessageTallyMock new. 
+
+	messageTally tally: 51.
+	model totalTime: 100.
+	
+	expected := model runtimeIcon bits.
+	actual := (model getIcon: messageTally) bits.
+	self assert: expected equals: actual
```

Comment:
I would insert a newline before the assert to visually separate the object construction from the actual test.